Source/WarpX.H

```diff
@@ -333,7 +333,7 @@ public:
     void ComputeMaxStep ();
     // Compute max_step automatically for simulations in a boosted frame.
     void computeMaxStepBoostAccelerator(const amrex::Geometry& geom);
-    int  MoveWindow (bool move_j);
+    int  MoveWindow (const int step, bool move_j);
```

Comment:
Also with regards to the `step`/`step+1` question we should define the meaning of the `step` here via a doxygen string